xml/System.Diagnostics/Process.xml

```diff
@@ -2530,7 +2530,7 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A process has a main window associated with it only if the process has a graphical interface. If the associated process does not have a main window (so that <xref:System.Diagnostics.Process.MainWindowHandle%2A> is zero), <xref:System.Diagnostics.Process.MainWindowTitle%2A> is an empty string (""). If you have just started a process and want to use its main window title, consider using the <xref:System.Diagnostics.Process.WaitForInputIdle%2A> method to allow the process to finish starting, ensuring that the main window handle has been created. Otherwise, the system throws an exception.  
+ A process has a main window associated with it only if the process has a graphical interface. If the associated process does not have a main window (so that <xref:System.Diagnostics.Process.MainWindowHandle%2A> is zero), <xref:System.Diagnostics.Process.MainWindowTitle%2A> is an empty string (""). On Unix platforms, this will always be an empty string (""). If you have just started a process and want to use its main window title, consider using the <xref:System.Diagnostics.Process.WaitForInputIdle%2A> method to allow the process to finish starting, ensuring that the main window handle has been created. Otherwise, the system throws an exception.  
```

Comment:
```suggestion
 A process has a main window associated with it only if the process has a graphical interface. If the associated process does not have a main window (so that <xref:System.Diagnostics.Process.MainWindowHandle%2A> is zero), or if the system can't determine that there's a main window (such as may be the case on some Unix platforms), <xref:System.Diagnostics.Process.MainWindowTitle%2A> is an empty string (""). If you have just started a process and want to use its main window title, consider using the <xref:System.Diagnostics.Process.WaitForInputIdle%2A> method to allow the process to finish starting, ensuring that the main window handle has been created. Otherwise, the system throws an exception.  
```